src/frontend.c

```diff
@@ -82,7 +90,210 @@ char *logfile_path = "out.txt";
 FILE *logfile = NULL;
 #endif
 
+// if version isn't defined by the Makefile
+#ifndef VERSION
+#define VERSION "unknown"
+#endif
+
+// cli pieces
+const char *program_name = "collascii";
+const char *program_version = VERSION;
+const char *program_bug_address = "<https://github.com/olin/collascii/issues/>";
+const char *program_description = "a collaborative ascii editor";
+const char *program_doc =
+    "COLLASCII\n\n"
+    "\"The Future Editor of Yesterday, Tomorrow!\"\n\n"
```

Comment:
Might be good to have a word here on starting a server (for someone who hasn't gone through extensive collascii training). Something like:

> To start a collascii server, make and run server.out with `make server.out && ./server.out`

Reply:
Would the README be a better place for that?

Reply:
That too. I'm thinking that someone who just downloaded collascii after skimming the Readme might do `--help` and like to see that info there.

Reply:
I added some changes the README and the CLI.